src/scripts/main.js

```diff
@@ -357,4 +357,45 @@ const people = [
 // eslint-disable-next-line no-console
 console.log(people); // you can remove it
 
-// write your code here
+const dashboard = document.querySelector('.dashboard');
+
+people.forEach((person) => {
+  const newString = document.createElement('tr');
```

Comment:
The variable name 'newString' is misleading because it suggests a string type, but it actually represents a table row element. Consider renaming it to something more descriptive, like 'row' or 'tableRow'.